views/index.html

```diff
@@ -53,6 +53,13 @@
         <div id="room-code" class="col">Game room 0</div>
         <div id="join-leave-updates" class="col"></div>
         <div id="deaths" class="col"></div>
+        <div class="sound-control">
+          <label class="sound-label" for="sound">Sound</label>
+          <input class="sound-checkbox" type="checkbox" id="sound" name="toggle-sound" value="" />
```

Comment:
Please replace all occurences of sound -> audio as a more standard way of refrence it. Please update the id to be more meaningful, for example, `audio-chkbox`. This is being referenced in the javascript file, so it needs to make it clear what is being referenced. Also, remove the attributes you don't need, like the value attribute which is empty.